packages/react-theme-provider/src/context.ts

```diff
@@ -0,0 +1,11 @@
+import { Theme } from '@fluentui/react-theme';
+import * as React from 'react';
+
+export interface ThemeProviderValue extends Theme {}
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export const internal__ThemeContext = React.createContext<ThemeProviderValue>(({} as unknown) as Theme);
```

Comment:
Seems like this go into the shared-contexts package. The context can never major bump, even if you want to major bump the Provider.

Reply:
I almost think maybe shared contexts should move to another repo just to avoid a major bumps in the future.

Major version updates end up duplicating the contexts leading to unexpected behaviors on the end user.

Reply:
An alternative I had thought of was to store all of the contexts globally so that you could say "if it's already on the page, use that". 

But that's a side effect and I would veer away from that if possible.

Reply:
> Major version updates end up duplicating the contexts leading to unexpected behaviors on the end user.

Even minor, it's extremely hard to debug such issues when contexts _are same, but not really same_.

---

It sounds like a good idea 👍 I named it as `internal__ThemeContext` to clarify that it's not Public API, opened for suggestions...